src/spikeinterface/core/recording_tools.py

```diff
@@ -910,3 +910,35 @@ def get_rec_attributes(recording):
         dtype=recording.get_dtype(),
     )
     return rec_attributes
+
+
+def check_recording_attributes_match(recording1, recording2_attributes, skip_properties=True):
+    """
+    Check if two recordings have the same attributes
+
+    Parameters
+    ----------
+    recording1 : BaseRecording
+        The first recording object
+    recording2 : BaseRecording
+        The second recording object
+
+    Returns
+    -------
+    bool
+        True if the recordings have the same attributes
+    """
+    recording1_attributes = get_rec_attributes(recording1)
+    recording1_attributes["probegroup"] = recording1.get_probegroup()
```

Comment:
Why do we need this line? It is never used that I can see?

Reply:
Or are we mutating in place? That seems dangerous.

Reply:
we don't need it anymore :)